src/XamarinBackgroundKit.Android/Renderers/MaterialBackgroundManager.cs

```diff
@@ -372,10 +377,27 @@ private void UpdateElevation()
             switch (_nativeView)
             {
                 case MaterialCardView mCardView:
-                    mCardView.SetElevation(_context, _backgroundElement);
+                    mCardView.SetElevation(_context, _backgroundElement);                   
                     break;
                 default:
-                    _nativeView.SetElevation(_context, _backgroundElement);
+                    _nativeView.SetElevation(_context, _backgroundElement);                    
+                    break;
+            }
+        }
+
+        private void UpdateShadowColor()
+        {
+            if (_nativeView == null || Build.VERSION.SdkInt < BuildVersionCodes.P) return;
+
+            switch (_nativeView)
+            {
+                case MaterialCardView mCardView:                   
```

Comment:
Since SetOutlineSpotShadowColor and SetOutlineAmbientShadowColor exist on View class, there is no need for separation